src/tests/ftest/osa/offline_reintegration.yaml

```diff
@@ -100,7 +100,6 @@ mdtest:
 test_obj_class:
   oclass:
     - RP_2G8
-    - RP_3G6
```

Comment:
You have to remove RP_2G8 since rf is set to 2.

Reply:
Arghhh, thanks for the tip.
Going to fix it and relaunch the test.